src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (value) => {
+    if (value === undefined) {
```

Comment:
You should explicitly check for `value` being strictly `undefined` using the strict equality operator (`===`). This is to avoid cases where `value` might be `null` or another falsy value that is not `undefined`, which should not trigger the return of the result.